### PR TITLE
Update translation.md

### PR DESCRIPTION
**Pull request by wangxy518**

### CODE REVIEW

File: translation.md [hunk 1]
```diff
@@ -27,13 +27,13 @@
...
...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     	| 操作员                                                                                                                                                                                                                                                                                                                                                                                                                                                                         	|        	|
```
Comment (idealhack) [inline translation.md]: Sorry, but I don't think these are good translations, these two are hard to translate.

File: translation.md [hunk 2]
```diff
@@ -11,7 +11,7 @@
...
...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       	| Chinese 中文                                                                                                                                                                                                                                                                                                                                                                                                                                                              	| Korean 韩文	|
```
Comment (idealhack) [inline translation.md]:
```suggestion
| English | Chinese 中文 | Korean 한국어 |
```

File: translation.md (hunk 1 above)
Comment (wangxy518) [reply translation.md]: hack在汉语里没有太合适的词，像之前的“骇客”，纯粹属于新造出来的词语。所以要用现有的词语，这样大概也说得过去。

**Comment by dankohn** [comment]
Thanks for proposing these changes. Our Chinese-speaking staff member agreed that these look good.

**Pull request merged**